tests/unit-tests/api/utilities/C_RemoveDirTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Beans\BeansFramework\Tests\API\Utilities;
+
+use Beans\BeansFramework\Tests\BeansTestCase;
+
+class RenderFunctionTest extends BeansTestCase {
+
+	protected $dir;
+
+	protected function setUp() {
+		parent::setUp();
+		$this->dir = dirname( __FILE__ ) . '/removeDir';
+		require_once BEANS_TESTS_LIB_DIR . '/api/utilities/functions.php';
+	}
+
+	protected function tearDown() {
+		parent::tearDown();
+		if ( is_dir( $this->dir ) ) {
+			if ( ! is_writable( $this->dir ) ) {
+				chmod( $this->dir, 0755 );
+			}
+			rmdir( $this->dir );
+		}
+	}
+
+	public function testBailsForNonDir() {
+		$dir = dirname( __FILE__ ) . '/';
```

Comment:
This can be `__DIR__`.

Reply:
@GaryJones Yes it can. `__DIR__` and `dirname(__FILE__)` are [equivalent to one another](http://php.net/manual/en/language.constants.predefined.php).